core/templates/newspapers.html

```diff
@@ -25,6 +25,7 @@
         <thead>
         <tr>
             <th class="headerSortDown left_no_border">State</th>
```

Comment:
The table sort is all JS, so it should just sort however the text would sort normally